.github/workflows/build.yml

```diff
@@ -95,11 +95,28 @@ jobs:
         #! This will also check that utilities programs still compile
         ./CI/scripts/test.sh test-run_example-simple_app
 
+    #! -------------------------------------------------------------------------
+    - name: test-simple_app-with-crypto_algorithms
+      run: |
+        #! ---------------------------------------------------------------------
+        # Set this env-var to indicate that we are running this test-case
+        # from CI, where previous test case has already run 'setup'.
+        # So, in this test case, we will not re-do 'setup'.
+        FROM_CI_BUILD_YML=1 ./CI/scripts/test.sh test-simple_app-with-crypto_algorithms
```

Comment:
Exercise the testing hook provided by `run_example.sh` to run `simple_app` with diff pairs of named crypto algorithms.